### PR TITLE
change default log level to error

### PR DESCRIPTION
**Pull request by wkloucek**
## Description
The default log level is now "error"

### CODE REVIEW

File: ocis-pkg/log/log.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func NewLogger(opts ...Option) Logger {
 		zerolog.SetGlobalLevel(zerolog.DebugLevel)
 		mlog.SetLevel(mlog.LevelTrace)
 	default:
-		zerolog.SetGlobalLevel(zerolog.InfoLevel)
+		zerolog.SetGlobalLevel(zerolog.ErrorLevel)
 		mlog.SetLevel(mlog.LevelInfo)
```
Comment (C0rby) [inline ocis-pkg/log/log.go:62]: Should this also be changed to the error level?

File: changelog/unreleased/change-default-log-level.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Change: change log level default from debug to error
+
+We've changed the default log level for all services from "default" to "error".
```
Comment (C0rby) [inline changelog/unreleased/change-default-log-level.md:3]:
```suggestion
We've changed the default log level for all services from "info" to "error".
```

File: ocis-pkg/log/log.go (hunk 1 above)
Comment (wkloucek) [reply ocis-pkg/log/log.go]: :+1:

**Review by C0rby — APPROVED** [review]
Cool :+1:

**Pull request merged**